docs/core/deploying/deploy-with-cli.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Publish .NET Core apps with the CLI
```

Comment:
Why the title suffix doesn't appear in the Google search ? It exists in the docfx.json

![image](https://user-images.githubusercontent.com/31348972/71553733-13090d80-2a1d-11ea-8b36-2057616e8a5e.png)

The title suffix appears in the browser within docs.microsoft.com without problem.

![image](https://user-images.githubusercontent.com/31348972/71553739-4b105080-2a1d-11ea-8565-9b074dcd4954.png)

Only doesn't appear in Google. So, I'm afraid my change here makes it appear like the following in Google:

"Publish apps with the CLI"

instead of the intended:

"Publish apps with the CLI - .NET Core"

Reply:
Interesting observation. I'm guessing that [Google](https://www.google.com/search?sxsrf=ACYBGNRgDOIdcvmvu4EUkRhEVqLyUYVmhA%3A1577732502114&ei=lkkKXvrDBsnq-gTaw6bQAQ&q=Publish+apps+with+the+CLI+-+.NET+Core&oq=Publish+apps+with+the+CLI+-+.NET+Core&gs_l=psy-ab.3..0i22i30.14460.14460..15321...0.4..0.75.75.1......0....2j1..gws-wiz.......0i71.6ktx7L-LHiE&ved=0ahUKEwj65trKh97mAhVJtZ4KHdqhCRoQ4dUDCAs&uact=5) chose to truncate the title to fit by excluding that part but including the last part "Microsoft Docs". On the other hand, [Bing](https://www.bing.com/search?q=Publish%20apps%20with%20the%20CLI%20-%20.NET%20Core&qs=n&form=QBRE&sp=-1&pq=publish%20apps%20with%20the%20cli%20-%20.net%20core&sc=1-37&sk=&cvid=EFB55E1A59984099BFBDC56B3105E825) chooses to truncate by excluding "Microsoft Docs" - which works better in this case.

Reply:
![image](https://user-images.githubusercontent.com/24882762/71596430-4bc7e600-2af4-11ea-9ce0-3812d71e9b23.png)

Reply:
I hope it only truncated the ".NET Core" part because it's **repeated**, that it doesn't truncate it after this PR.

Reply:
Ahh yes, you might be right, and I hope that's the case too.